src/app/containers/app/app.component.ts

```diff
@@ -17,15 +19,30 @@ export class AppComponent implements OnInit {
   public timeoutModalConfig = {
     countdown: '0 seconds',
     isVisible: false,
-  }
+  };
 
   constructor(
     private readonly store: Store<fromRoot.State>,
-    private googleAnalyticsService: GoogleAnalyticsService,
-    private readonly timeoutNotificationsService: TimeoutNotificationsService
+    private readonly googleAnalyticsService: GoogleAnalyticsService,
+    private readonly timeoutNotificationsService: TimeoutNotificationsService,
+    private readonly router: Router,
+    private readonly titleService: Title
   ) {
 
     this.googleAnalyticsService.init(config.googleAnalyticsKey);
+
+    this.router.events.subscribe((data) => {
```

Comment:
Maybe assign to Subscription so you can unsubscribe in onDestroy?

Reply:
No need as this is the appcomponent, which lives throughout the whole application livecycle. If it was a deeper component yes, but here it's not necessary, so better from a memory point of view to not keep that reference around.